### DISCUSSION TITLE
How to use @EnableDynamoDBRepositories using sdk2?

### DISCUSSION THREAD

**Discussion by dcj2489** [aws/aws-sdk-java-v2#4645] [Migration from v1 to v2]
I'm migrating from sdk1 to sdk2, however, I used @EnableDynamoDBRepositories before, but now with sdk2, the annotation is not creating the beans and also I noticed `DynamoDBMapperConfig` does not exist.

```
   @Bean
    public DynamoDBMapperConfig dynamoDBMapperConfig(DynamoDBMapperConfig.TableNameOverride tableNameOverrider) {
        DynamoDBMapperConfig.Builder builder = new DynamoDBMapperConfig.Builder();
        builder.withTypeConverterFactory(DynamoDBTypeConverterFactory.standard());
        builder.withTableNameResolver(DefaultTableNameResolver.INSTANCE);
        builder.withTableNameOverride(tableNameOverrider());
        return builder.build();
    }
```

**Comment by debora-ito** [comment]
A quick internet search showed that `@EnableDynamoDBRepositories` is a Spring Data annotation, so you'll have to ask the Spring Data support folks about this.

For `DynamoDBMapperConfig`, we don't have an equivalent class in V2, but we have a tracking feature request to support the table name override - https://github.com/aws/aws-sdk-java-v2/issues/2935. Feel free to add a thumbs up and a comment about your use case.
